### PR TITLE
[16.0][MIG] edi_endpoint_oca

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Standard migration.

Todo:
* [ ] Functional review
* [ ] Review automated tests

### CODE REVIEW

File: edi_endpoint_oca/models/__init__.py [hunk 1]
```diff
@@ -2,4 +2,3 @@
 from . import edi_backend_type
 from . import edi_endpoint
 from . import edi_exchange_record
-from . import edi_exchange_consumer_mixin
```
Comment (simahawk) [inline edi_endpoint_oca/models/__init__.py]: WHY?

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by LoisRForgeFlow** [comment]
rebased

**Comment by thienvh332** [comment]
LGTM
@JordiBForgeFlow Pre-commit is not happy. Could you please take a look.

**Comment by simahawk** [comment]
@JordiBForgeFlow @LoisRForgeFlow could you please have a look at #46 ?

**Pull request closed without merging**